Cargo.toml

```diff
@@ -117,7 +117,7 @@ version_check = "0.9.5"
 watchman_client = { version = "0.9.0" }
 whoami = "1.6.0"
 winreg = "0.52"
-
+winapi= {version = "0.3.9", features = ["winuser"] }
```

Comment:
Do we need the `winuser` feature? I don't even know what it's for; I'm just checking to make sure because I didn't see any mention of "user" in the code.